csharp/OnnxRuntime.CSharp.proj

```diff
@@ -144,6 +147,87 @@ CMake creates a target to this project
     />
   </Target>
 
+  <!-- To support build.py and CI easily, generate the sub packages in the proj -->
+  <Target Name="CreateSubGpuWinPackages" AfterTargets="CreatePackage" Condition="'$(OrtPackageId)' == 'Microsoft.ML.OnnxRuntime.Gpu'">
```

Comment:
Instead of cut-and-pasting can we define properties for the values that change each time (may just be package name) and use CallTarget?

e.g. something like this: https://stackoverflow.com/a/2017647/684911

Reply:
Thank your suggestion. updated